public/locales/en/translation.json

```
@@ -221,6 +232,20 @@
             "reparing": "Error Repairing"
         },
         "finished": {
+            "add": {
+                "steam": {
+                    "corrupt": "{{game}} could not be added to all found Steam users. See logs for more info.",
+                    "success": "{{game}} was successfully added to Steam.",
+                    "title": "Added to Steam"
+                }
+            },
+            "remove": {
+                "steam": {
+                    "body1": "{{game}} was successfully removed to Steam.",
+                    "body2": "{{game}}  could not be removed from all found Steam users. See logs for more info.",
```

```suggestion
                    "body2": "{{game}} could not be removed from all found Steam users. See logs for more info.",
```

I think it should be fixed in all files.